tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp

```diff
@@ -3442,4 +3442,52 @@ namespace SimulationTests
 			}
 		}
 	};
+
+	public ref class when_running_simulation_with_ph_solubility_table : public when_running_pksim_input
+	{
+	protected:
+		virtual void Because() override
+		{
+			when_running_pksim_input::Because();
+
+			_inputFile = "pH_Solubility_Table";
+			_venPlsId = "tH4ucab_aUib7yTJoRIUFg";
+			_writeDebugFile = false;
+			_writeMatlabCode = false;
+			_writeCppCode = true;
+			_exportFullMode = true;
+		}
+
+	public:
+		[TestAttribute]
+		[Ignore("TODO")]
+		void should_perform_simulation_run()
```

Comment:
set it to ignore, otherwise it will run for minutes and crash.
This is a simulation with "real" solubility table